website/jscripts/signature.js

```diff
@@ -30,6 +30,14 @@ document.addEventListener("DOMContentLoaded", function() {
                 const sigUrls = postWrapper.querySelector('.sig_urls');
                 if (sigUrls && !sigUrls.hasAttribute('data-user-set-color')) {
                     sigUrls.style.color = usernameColor;
+
+                    // Apply the color to .sig_urls a and a:hover elements
+                    const style = document.createElement('style');
```

Comment:
Why do we create styles via javascript instead of declaring them in the templates?

Reply:
![image](https://github.com/user-attachments/assets/5d083913-695b-4e9f-bd3f-f477c2d3fd14)

Reply:
@Irysek707 When I was making that comment I wasn't understanding well what you were trying to achieve here, but after spending another moment I understood why it is the way you did it.
I would very much like the idea of not stuffing hundreds of events into DOMContentLoaded, but in this specific case - I do not have an easy idea on how to solve it otherwise. So it can stay that way for now.

I added a small commit - I changed your usages of console.log to console.debug - so spamming of the console will happen only when you set level to verbose. It's better practice if we're not logging anything important.